08-rails/db-seeds.md

```diff
@@ -14,22 +14,26 @@ Rails applications come pre-installed with an empty seed file, `db/seeds.rb`. On
 
 The seed file is a "normal" ruby file, and it has access to the Model objects that we set up in our application. If we want to create a number of seeded objects, we can create a list with the properties for each object, then iterate over it, creating objects as we go.
 
-This is an example which assumes that we have already created a `Student` model object.
+This is a simplified example which assumes that we have already created an `Author` model object which has a `name` and a `bio` field in the schema.
 ```ruby
 # db/seeds.rb
-students = [
-  { first_name: "Ada",
-    last_name: "Lovelace",
-    cohort: 29
+authors = [
+  {
+    name: "Margot Lee Shetterly",
+    bio_url: "https://en.wikipedia.org/wiki/Roxane_Gay"
   },
-  { first_name: "Grace",
-    last_name: "Hopper",
-    cohort: 29
+  {
+    name: "Sandi Metz",
+    bio_url: "https://en.wikipedia.org/wiki/Sandi_Metz"
+  },
+  {
+    name: "Octavia E. Butler",
+    bio_url: "https://en.wikipedia.org/wiki/Octavia_E._Butler"
   }
 ]
 
-students.each do |student|
-  Student.create(student)
+authors.each do |author|
+  Author.create(author)
 end
 ```
```

Comment:
I like how we're not explicitly doing CSV here and forcing them to put together two pieces of learning in the project.